app/assets/stylesheets/shared/form.scss

```diff
@@ -86,8 +86,12 @@
     border-radius: $input-border-radius;
   }
 
-  .ss-disabled ~ label:before {
-    background: transparent;
+  [disabled="disabled"],
```

Comment:
This is what we needed for all the labels. It will make all disabled labels have a gray bg.